opentelemetry-sdk/src/opentelemetry/sdk/metrics/view.py

```diff
@@ -68,36 +68,64 @@ class View:
     def __init__(
         self,
         metric: InstrumentT,
-        aggregator: Aggregator,
+        aggregator: type,
+        aggregator_config: dict = None,
         label_keys: Sequence[str] = None,
-        config: ViewConfig = ViewConfig.UNGROUPED,
+        view_config: ViewConfig = ViewConfig.UNGROUPED,
```

Comment:
What do you think about Chris' comments about config?

>Another approach that might simplify things -- and avoid the oddball case in the example where label_keys is ignored -- is to get rid of this class and treat null label_keys like UNGROUPED (or better, use a sentinel as the default arg value like CURRENT_SPAN) and treat an empty list as DROP_ALL.

Reply:
It's definitely a reasonable solution, but in the interest of not having to update the code more I'd prefer to leave this until future PRs. It's also much less explicit which might make things even more confusing